CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+# Contributing to Lisk-JS
+
+First off, thanks for taking the time to contribute! :raised_hands:
+
+The following is a set of guidelines for contributing to Lisk-JS, which are hosted in the [LiskHQ Organization](https://github.com/LiskHQ) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[I don't want to read this whole thing, I just have a question!!!](#i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
```

Comment:
Help! I don't want to read this whole thing, I just have one question. :mag_right: